pepper-apis/src/main/java/org/broadinstitute/ddp/route/PatchFormAnswersRoute.java

```diff
@@ -212,9 +211,12 @@ public Object handle(Request request, Response response) {
                             Set<Long> answerIds = answerDao.getAnswerSql()
                                     .findAnswerIdsByInstanceGuidAndQuestionId(instanceGuid, questionDto.getId());
                             if (answerIds.size() > 1) {
-                                String msg = "Question is already answered. Provide the answer guid to update.";
-                                LOG.info(msg);
-                                throw ResponseUtil.haltError(response, 409, new AnswerExistsError(msg, questionStableId));
+                                String errMsg = String.format(
+                                        "A question can have 1 and only 1 answer but found %d answers instead",
```

Comment:
`can have 1 and only 1 answer` sounds a little too "strong"? Maybe try this?
```
"Question %s is expected to have one answer but found %d instead", questionStableId, answerIds.size()
```

Reply:
Done.

Reply:
Can we remove the `A` at the start? `A question XYZ ...` doesn't sound right.

Reply:
Yeah, the combination of the `A` article and a `GUID` that implies a specific question aren't compatible. Removed.